Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -43,21 +41,24 @@ private func toJsonFieldName(_ s: String) -> String {
 /// string data.  Track the buffers and release all of those buffers
 /// in case we ever get deallocated.
 fileprivate class InternPool {
-  private var interned = Set<UnsafePointer<UInt8>>()
+  private var interned = [UnsafeBufferPointer<UInt8>]()
 
   func intern(utf8: String.UTF8View) -> UnsafeBufferPointer<UInt8> {
     let count = utf8.count
-    let raw = malloc(count)!
+    let raw = UnsafeMutableRawPointer.allocate(bytes: count, alignedTo: 1)
```

Comment:
Could some of this be collapsed with https://developer.apple.com/reference/swift/unsafemutablepointer/2295026-allocate?

```swift
let buff = UnsafeMutablePointer<UInt8>.allocate(capacity: count)
_ = buff.initialize(from: utf8)
```

?